sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please specify the release date here by replacing `Unreleased`, the format is `yyyy-mm-dd`

Reply:
Done.